### DISCUSSION TITLE
ECG data (SCP-ECG file)

### DISCUSSION THREAD

**Discussion by tGitm** [nextgenhealthcare/connect#5104] [Q&A]
Is it possible to get raw ECG data (SCP file)?
This is all I found on old Mirth forums: [https://forums.mirthproject.io/forum/mirth-connect/general-discussion/8090-dicom-waveform-scp-ecg-ecg-support]( https://forums.mirthproject.io/forum/mirth-connect/general-discussion/8090-dicom-waveform-scp-ecg-ecg-support)

**Comment by janmarc** [comment]
Yes we would have the same question, how to process RAW data from any device, including EEG and ECG.
